### PR TITLE
[aptos-cli] Add faucet option for creating account

### PR DESCRIPTION
**Pull request by kent-white**
## Motivation

#470. Currently for the `account create` command you need another funded account to sign the transaction to create a new account. This would be frustrating for someone trying to make their first account. It would be more convenient to just use the faucet.

## Test Plan

```
$ aptos_ account create 6CC3B2D302EFEA6BFE0D4B674CABF0A3849EB6ACF421F960DD5D73773ADE3C32 9                   
Unexpected error: One of ['--private-key', '--private-key-file', '--use-fauet'] must be provided


$ aptos_ account create 6CC3B2D302EFEA6BFE0D4B674CABF0A3849EB6ACF421F960DD5D73773ADE3C32 9 --use-faucet      
Account Created at B8ACE07BF9DD3ADF0107BF29A385A5838256A85838403546CA9B7417B747F67F
```

## Related PRs

#470

### CODE REVIEW

File: crates/aptos/src/account/create.rs [hunk 1]
```diff
@@ -99,26 +104,60 @@ impl CreateAccount {
...
+            .send()
+            .await?;
+        if response.status() == 200 {
+            Ok(response.status().to_string())
+        } else {
+            Err(Error::new(CommonError::UnexpectedError(format!(
+                "Faucet issue: {}",
+                response.status()
+            ))))
+        }
```
Comment (gregnazario) [inline crates/aptos/src/account/create.rs:124]: btw, there is a `FaucetClient` so you can probably just say `FaucetClient::new(account: AccountAddress)`

**Review by gregnazario — APPROVED** [review]
I'm not really concerned about anything here.  I can clean some stuff up afterwards if you're moving onto something else.

File: crates/aptos/src/account/create.rs (hunk 1 above)
Comment (kent-white) [reply crates/aptos/src/account/create.rs:124]: I tried it, but it's blocking which doesn't play nice with command line. Would have to refactor FaucetClient a little I think.

**Pull request closed without merging**